css/index.css

```diff
@@ -101,17 +129,107 @@ p {
     justify-content: space-evenly;
 }
 
```

Comment:
You should make use of parent/child relationships with the individual boxes.  The .middle-content .boxes .box contains all the styling for each box except for the color.  See my comments in the index.hml file as well.